app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,61 @@
+class TasksController < ApplicationController
+  before_action :set_task, only: [:edit, :show]
+  before_action :set_task_update, only: [:update, :destroy]
+
+  def index
+    if current_employee.role == Employee::ADMIN_ROLE
+      @project = current_employee.company.projects.find(params[:id])
+    end
+  end
+
+  def new
+    @task = current_employee.company.projects.find(params[:id]).tasks.new
+  end
+
+  def create
+    @task = Task.new(tasks_create_params)
+    @task.set_status
+    @task.set_assignable(params[:assignable_team_id], params[:assignable_employee_id], params[:task][:assignable_type])
```

Comment:
check if params[:task] is present

Reply:
That is present but these are like hidden ones. The controller does the job based on this.